etc/linux-systemd/system/syncthing-relay.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Syncthing - Relay
+After=network.target
+ 
+[Service]
+User=relaysrv
+ExecStart=/usr/local/bin/relaysrv -keys /usr/local/etc/relaysrv/ -pools ""
+Restart=on-failure
+SuccessExitStatus=3 4
+RestartForceExitStatus=3 4
```

Comment:
So we need to remove the 3 last lines ?
I don't know systemd well I only adapt the script for syncthing.